### DISCUSSION TITLE
[@types/twitch-ext] BitsTransaction.initiator in lowercase

### DISCUSSION THREAD

**Discussion by gdamou** [DefinitelyTyped/DefinitelyTyped#67555] [Issues with a @types package]
Hi,

I am currently working with the @types/twitch-ext package and I have noticed that the type for the interface BitsTransaction.initiator is "CURRENT_USER" | "OTHER".

On the twitch documentation it is the same but when you get the bits transaction object you can see that the type of initiator is in lowercase.

![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/46034621/a6cf8879-1f2f-447b-8651-000f7942519a)
![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/46034621/618e4257-7479-4461-9aa4-3090da6cdbf4)

[a comment by an automated account is not shown]

**Comment by demensky** [comment]
I think the Twitch developers need to be notified about this. This is probably best done on the [Twitch Developer Forums](https://discuss.dev.twitch.com/).

**Reply by gdamou** [reply]
Yeah you are right, just made a topic on the twitch developer forum, will keep in track the discussion to when they respond and/or change the doc.

topic : https://discuss.dev.twitch.com/t/bitstransaction-initiator-in-lowercase-instead-of-uppercase/51813

**Reply by beheh** [reply]
I see this is now tracked at twitchdev/issues#873.

**Comment by demensky** [comment]
Done #67557
